### ISSUE TITLE
3 way merge driver usage

### ISSUE THREAD

**Issue by burner1024** [WeblateOrg/weblate#9328]
### Describe the issue

Hi.
I've been looking for ways to reduce merge conflicts.
This [issue](#55) and [this file](https://github.com/WeblateOrg/weblate/blob/8c1e02d3ec981d70bd9a19ac23c66bcbc6da35f0/weblate/examples/git-merge-gettext-po#L3) came up.
I can't find any documentation about it.
The comment in the file says that it's installed for all weblate internal repositories. What are "internal repositories"? Just Weblate's local copies of upstream, or something else? I checked git config in weblate container, and the setting is not there.

Is it already used in my installation, and I just don't know about it? Or it specifically needs to be enabled? If so, can it be enabled globally for all repos?

### I already tried

- [X] I've read and searched [the documentation](https://docs.weblate.org/).
- [X] I've searched for similar issues in this repository.

### Steps to reproduce the behavior

-

### How do you run Weblate?

Docker container

**Comment by nijel** [comment]
It is installed in `DATA_DIR/home/.config/git/attributes` and `DATA_DIR/home/.gitconfig`, see https://docs.weblate.org/en/latest/admin/config.html#std-setting-DATA_DIR

**Comment by burner1024** [comment]
So, it is used by default for all repos, and no additional configuration is required?

**Comment by nijel** [comment]
Yes, it is used. All commands in Weblate are executed with `HOME=DATA_DIR/home`.

**Comment by burner1024** [comment]
I see, thank you for the explanation!
